### ISSUE TITLE
Satellite UI: detailed usage reports

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#6154]
There should be a way in the UI to download a CSV or PDF of detailed usage for a given period of time with information like project name, bucket name, storage, and egress.

We already have the backend code necessary to get the data. We just need to convert it to a csv file and add something to the UI to allow you to download it for a given period of time.

### AC
* I can download a detailed usage report from the satellite UI

**Comment by ferristocrat** [comment]
@boshevski - do you have a design for this?

**Comment by boshevski** [comment]
Yes, i updated the vuetify repo with designs to download usage reports for all project and per project, just like we discussed in design/dev sync.

**Comment by ferristocrat** [comment]
> Yes, i updated the vuetify repo with designs to download usage reports for all project and per project, just like we discussed in design/dev sync.

Added the link and screenshot to the description above

**Comment by iglesiasbrandon** [comment]
i cant wait for this report!
